SIS_import/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  job:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    volumes:
+      - ${HOME}/secrets/canvas
```

Comment:
It doesn't look like this volume is being used. Is this for a future fix?

Reply:
It is not used. I will remove it